{"uuid":"f2513110-30ed-4275-a837-a467cc800984","name":"Scenario Outline: The user getting error message with invalid python code for Introduction page","children":["74375130-d247-460a-84fa-a5dc016d4886"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884941427,"stop":1696884941427},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884941504,"stop":1696884941504},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884941609,"stop":1696884941610},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884943668,"stop":1696884943668}],"start":1696884941078,"stop":1696884943669}